tests/topotests/multicast_pim_uplink_topo3/test_multicast_pim_uplink_topo3.py

```diff
@@ -0,0 +1,920 @@
+#!/usr/bin/env python3
+#
+# SPDX-License-Identifier: ISC
+#
+# Copyright (c) 2023 by VMware, Inc. ("VMware")
+#
+
+"""
+Following tests are covered to test multicast pim sm:
+
+1. TC:1 Verify static IGMP group populated when static "ip igmp join <grp>" in configured
+2. TC:2 Verify mroute and upstream populated with correct OIL/IIF with static igmp join
+3. TC:3 Verify local IGMP join not allowed for "224.0.0.0/24" and non multicast group
+4. TC:4 Verify static IGMP group removed from DUT while removing "ip igmp join" CLI
+5. TC:5 Verify static IGMP groups after removing and adding IGMP config
+"""
+
+import os
+import re
+import sys
+import json
+import time
+import datetime
+from time import sleep
```

Comment:
Where is this used?

Reply:
Hi @ton31337 Thanks for reviewing PR, it was left, removed now.